src/App.jsx

```diff
@@ -1,34 +1,23 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import './App.css'
-
+import Nav from './components/Nav/Nav'
+import Home from './components/home/home'
+import Error from './components/Error/Error'
```

Comment:
Must: The casing on the filenames here is inconsistent, please could the Home import be updated to match the others?